src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs

```diff
@@ -559,6 +559,9 @@ public DisplayVersion SelectedVersion
 
                         NuGetUIThreadHelper.JoinableTaskFactory.RunAsync(() => SelectedVersionChangedAsync(_searchResultPackage, _selectedVersion.Version, loadCts.Token).AsTask());
                     }
+
+                    OnPropertyChanged(nameof(SelectedVersion));
```

Comment:
Since `SelectedVersion` will now fire before `PackageMetadata` fires, was there anything relying on (listening to) `SelectedVersion` directly that now may become stale?

eg, Is `CanInstall` set appropriately, and the state of the Install button kept in sync? Can I click Install before the selected version gets bound?

eg, Combobox  `_versions` selected value is bound to this property. Does it remain unset forever, or some other logic eventually sets SelectedItem?

Reply:
I didn't notice anything that this would cause issues with.

I tested with a known package that takes a long time to display the package metadata 'test10k', I was able to install/uninstall the package while it was still loading the metadata and all the icons that I expected to update were updated correctly.

Was there a specific scenario you think I should test?  I would be more than happy to walk through a video call with you to test some things out if you would prefer.

Reply:
Nope, sounds like you tested the install/uninstall. Just making sure it was covered.